### ISSUE TITLE
Issue starting multiple servers in 1.4.3

### ISSUE THREAD

**Issue by aaronrenner** [edeliver/edeliver#226]
### Environment

* Edeliver version (`mix edeliver --version`): 1.4.3
* Elixir version (`elixir -v`): 1.4.2
* Build tool and version (distillery|exrm): distillery
* Operating system (on build / deploy hosts): ubuntu 14.04
* Are you using an umbrella project (yes|no): yes

### Verbose output

* Are there more information when running with `--verbose`? No
* Can you narrow down the code line causing the issue when running with `--debug`?
* Please remove any private data from the output before pasting it here (login credentials, ip addresses user names etc.)

### Current behavior

Include executed edeliver command, error output etc.

```
$ mix edeliver start

EDELIVER ISSUER WITH START COMMAND

-----> starting staging servers

Permission denied, please try again.
Permission denied, please try again.
Permission denied (publickey,password).

staging node: 1

  user    : app2.myapp.vagrant:~
  host    : app2.myapp.vagrant
  path    : ~
  response: failed



staging node: 0

  user    : _myapp-api
  host    : app1.myapp.vagrant
  path    : app1.myapp.vagrant
  response: failed
```

Here are my staging variables from `.deliver/config`.

```
STAGING_USER="_myapp-api"
STAGING_HOSTS="app1.myapp.vagrant app2.myapp.vagrant"
```

It does work fine when running with 1 server. 

### Expected behavior

The expected behavior is for it to start correctly. Here is the output in version 1.4.2

```
$ mix edeliver start

EDELIVER ISSUER WITH START COMMAND

-----> starting staging servers


staging node: 0

  user    : _myapp-api
  host    : app1.myapp.vagrant
  path    : ~
  response:


staging node: 1

  user    : _myapp-api
  host    : app2.myapp.vagrant
  path    : ~
  response:
```
The path, user and host are all correct when using edeliver 1.4.2.

**Comment by benrom** [comment]
Same kind of problem here (debian 8, 2 hosts).

My _permission denied_ error is slightly different:
> Permission denied (publickey).

I can connect by ssh to those hosts without any problem.

**Comment by geofflane** [comment]
👍  Seeing this on Edeliver on 1.4.3 as well. Maybe related to the "spaces in paths" change?

**Comment by prashanthsadasivan** [comment]
I think this is still happening, any word on whether this will be fixed or not?

**Comment by brucepom** [comment]
To add a little detail, this seems to only affect some Edeliver commands. For example, this command works fine:
`mix edeliver deploy release to production`
seeming to indicate that Edeliver _can_ connect to my web servers over ssh and successfully deploy files to them. However the following command fails, apparently with this command Edeliver does not successfully make an ssh connection to my web servers:
```
$ mix edeliver restart production

EDELIVER MTAPP WITH RESTART COMMAND

-----> restarting production servers

Permission denied (publickey).
```

**Comment by darksheik** [comment]
The deploy command successfully deploys, but trying to run the `start` task or any custom task fails and does not appear to be assembling the `user` and `path` correctly.  It works correctly if only a single node is configured, but then multiple nodes produce this behavior:

```
production node: 1

  user    : 10.10.1.0:/home/ubuntu/web
  host    : 10.10.1.0
  path    : /home/ubuntu/web
  response: failed



production node: 0

  user    : ubuntu
  host    : 10.10.1.1
  path    : 10.10.1.1
  response: failed
```

**Comment by overture8** [comment]
Yeah, I'm getting this too. The `build` and `deploy` commands work fine - but `start` and `ping` commands seem to be using an incorrect path/user.

**Comment by bharendt** [comment]
Thanks for reporting and investigating this! We reverted the 958ffbb commit and published version [1.4.4](https://hex.pm/packages/edeliver/1.4.4). Would be nice if you could confirm, that it works again for you.

**Comment by darksheik** [comment]
@bharendt Tried `1.4.4` and the multi-node tasks look good now.  Thanks!
